src/ray/util/process.cc

```diff
@@ -407,7 +407,8 @@ std::string Process::Exec(const std::string command) {
 #ifdef _WIN32
   std::unique_ptr<FILE, decltype(&_pclose)> pipe(_popen(command.c_str(), "r"), _pclose);
 #else
-  std::unique_ptr<FILE, decltype(&pclose)> pipe(popen(command.c_str(), "r"), pclose);
+  std::unique_ptr<FILE, void (*)(FILE *)> pipe(popen(command.c_str(), "r"),
+                                               [](FILE *f) -> void { pclose(f); });
```

Comment:
Can you try https://code.videolan.org/videolan/vlc/-/merge_requests/4057/diffs?commit_id=7cf5d6977a07593a1b7cdb163b0748e6e60ee600. This might fix the issue without lambda.

Reply:
Looks like this build issue is fixed, but now I'm getting something else:

```
    bazel-out/k8-opt/bin/external/com_github_spdlog/_virtual_includes/spdlog/spdlog/fmt/bundled/core.h:1735:15: error: possibly dangling reference to a temporary [-Werror=dangling-reference]
     1735 |   const auto& arg = arg_mapper<Context>().map(FMT_FORWARD(val));
```

I'll file a separate issue.